src/index.html

```diff
@@ -17,6 +17,87 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="link__logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo"
```

Comment:
The 'alt' attribute should describe the image content. In your case, 'logo' is not descriptive enough. Try to provide more specific description. For instance, if it's a logo of a company, you can specify the company name and what kind of business it does. This will help improve your page's SEO and accessibility.